test/Feature/StructureSpec.hs

```diff
@@ -223,11 +223,18 @@ spec = do
 
       it "includes body schema for arguments" $ do
```

Comment:
`includes function summary/description and body schema for arguments` would be more descriptive, now that you added checks for summary/description.